app/forms/course_detail_form.rb

```diff
@@ -87,6 +91,29 @@ def update_trainee
     end
   end
 
+  def reassign_autocomplete_errors
```

Comment:
We could dry this up into a helper to share across all the form objects, so it isn't *that* bad.